dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/DotTemplateTool.java

```diff
@@ -11,6 +11,7 @@
 import com.dotcms.contenttype.transform.JsonTransformer;
 
 import org.apache.velocity.context.Context;
+import org.apache.velocity.tools.view.context.ChainedContext;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.velocity.tools.view.context.ChainedContext'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)